qiling/extensions/coverage/formats/history.py

```diff
@@ -122,13 +132,13 @@ def get_mem_map_from_addr(self, ins: int) -> tuple:
         # i sure hope theres not more than one map that contains the instruction lol
         return mem_map[0]
 
-    def get_regex_matching_exec_maps(self, libs: List) -> List:
+    def get_regex_matching_exec_maps(self, libs: List[str]) -> List:
```

Comment:
It looks like `libs` should be of type `Union[str, Collection[str]]` as it accepts both options.